docs: add weekly report 0228 (#932)
iresty/iresty.github.io: website/blog/2022/03/03/weekly-report-0228.md

---

> title: Biweekly Report｜2.15-2.28 Feature Highlights Update in Progress

Reviewer: `Feature Highlights Update in Progress` 😂 Cannot get your point

Reviewer: Refer to:

![image](https://user-images.githubusercontent.com/2106987/156679541-4db4d636-5b64-40ae-ad7a-50b51d8c3a2b.png)

Author: How about this: Apache APISIX Community Explore in Feb.15 - Feb.28

Reviewer: How about `Explore the week of Feb 15 - Feb 28`? @yzeng25 @lijing-21 If it starts with `Apache APISIX`, it's too long.

Reviewer: ```suggestion
title: Biweekly Report｜Explore the week of Feb 15 - Feb 28
```

Author: @juzhiyuan @yzeng25 Maybe `Explore the two weeks of Feb 15 - Feb 28` and `社区双周报 | 2.15 - 2.28` for Chinese ver?

---
title: Biweekly Report｜2.15-2.28 Feature Highlights Update in Progress
keywords:

> - Apache APISIX
> - Weekly Report
> - Contributor
> - APISIX
> - API Gateway
> - Apache

Reviewer 2: ```suggestion
- Apache APISIX
- API Gateway
- Weekly Report
- Contributor
```

## Good first issue

### Issue #920

**Link**: https://github.com/apache/apisix-website/issues/920

**Issue description**:

When submitting a pull request to add a new blog to the website's blog section, usually we need to submit 2 files, one in English and the other in Chinese.

As the community grows, blog in both languages are becoming more diverging than before. Thus, writing and submitting blogs in single-language is the trend.

But the actual behavior is that: when submitting an English blog, and clicking the language conversion button on the top-right corner, it would jump to the default 404 page. I am wondering if there is any chance to enhance the reading experience on this part.

### Issue #6460

**Link**: https://github.com/apache/apisix/issues/6460

**Issue description**:

Using the `authz-keycloak` plugin when access is not permitted you correctly receive an access denied message in the body of the requested url.

```Json
{"error":"access_denied","error_description":"not_authorized"}
```

Is it possible to specify a redirect url to be used when the access is denied so the user see predefined page instead of a message?

## Highlights of Recent Features

> - [Push access log to clickhouse DB](https://github.com/apache/apisix/pull/6215)（Contributor: [zhendongcmss](https://github.com/zhendongcmss)）

Reviewer 2: ```suggestion
- [Push access log to clickhouse DB](https://github.com/apache/apisix/pull/6215)(Contributor: [zhendongcmss](https://github.com/zhendongcmss)(
```

> - [Add Kubernetes discovery module](https://github.com/apache/apisix/pull/4880)（Contributor: [zhixiongdu027](https://github.com/zhixiongdu027)）

Reviewer 2: ```suggestion
- [Add Kubernetes discovery module](https://github.com/apache/apisix/pull/4880)(Contributor: [zhixiongdu027](https://github.com/zhixiongdu027))
```

> - [Add config to control return all status at X-APISIX-Upstream-Status](https://github.com/apache/apisix/pull/6392)（Contributor: [liangliang4ward](https://github.com/liangliang4ward)）

Reviewer 2: ditto

> - [Add `mocking` plugin](https://github.com/apache/apisix/pull/5940)（Contributor: [Drery](https://github.com/Drery)）

Reviewer 2: ```suggestion
- [Add `mocking` plugin](https://github.com/apache/apisix/pull/5940)(Contributor: [Drery](https://github.com/Drery))
```

The Apache APISIX project website and the Github issue have accumulated a wealth of documentation and experience, so if you encounter problems, you can read the documentation, search the issue with keywords, or participate in the discussion on the issue to put forward your own ideas and practical experience.

## Recent Blog Recommendations

- [The practice of Nacos service discovery on API Gateway](https://apisix.apache.org/blog/2022/02/21/nacos-api-gateway)：

  This article introduces the basic concepts of Apache APISIX and Nacos and Service Registry, and shows you the practice of Nacos service discovery on API Gateway.

> - [Apache APISIX Enhances API Security by CSRF Plugin](https://apisix.apache.org/blog/2022/02/23/csrf)：

Reviewer 2: ```suggestion
- [Apache APISIX Enhances API Security by CSRF Plugin](https://apisix.apache.org/blog/2022/02/23/csrf-api-gateway):
```

  This article introduces `csrf`, the CSRF security plugin for Apache APISIX, and details how to secure your API information in Apache APISIX with the help of the `csrf` plugin.

> - [How to Integrate API Gateway and Consul?](https://apisix.apache.org/blog/2022/02/25/consul-api-gateway)：

Reviewer 2: ```suggestion
- [How to Integrate API Gateway and Consul?](https://apisix.apache.org/blog/2022/02/25/consul-api-gateway):
```
